poc_iot_injector/src/receipt_txn.rs

```diff
@@ -142,10 +144,10 @@ fn construct_poc_witnesses(
             let poc_witness = BlockchainPocWitnessV1 {
                 gateway: witness_report.report.pub_key.into(),
                 timestamp: witness_report.report.timestamp.timestamp() as u64,
-                signal: witness_report.report.signal,
+                signal: witness_report.report.signal / SIGNAL_MULTIPLIER,
```

Comment:
i assume this will always cleanly divide into a whole number without underflow or fractional components?

Reply:
I believe it should, from what I could find the incoming signal and snr are both `*10` so this is just dividing that.

Reply:
Err, it's i32/i32 but the info after decimal point will be lost, yes. But I think that's okay? So in the witness_report both snr and signal are i32 but for the chain, snr is f32 and signal is i32 (in blockchain_poc_witness_v1) . The more critical bit is that both of these are multiplied by a factor of 10 when they come in, so at the very last step we do the div by 10.

Still, if you guys think we should add some protection around the division, please to be suggesting.

Reply:
I'm more worried about the div 10 losing information that should really be available to chain? If the div loss is expected, I'm all good